lib/kwtool_web/controllers/auth_controller.ex

```diff
@@ -0,0 +1,9 @@
+defmodule KwtoolWeb.AuthController do
+  use KwtoolWeb, :controller
+
+  plug :put_layout, "auth.html"
+
+  def sign_up(conn, _params) do
```

Comment:
In terms of controller method naming, prefer to follow CRUD operations (new, create, show, edit, delete) as much as possible for consistency.

Reply:
I updated it.